### PR TITLE
Improving function signatures (#98)

### LINKED ISSUE #98: Document lang.Translate template function
Currently, it's not documented on it's own.

/cc @bep - are `i18n` and `T` just aliases for `lang.Translate`?

### PR DESCRIPTION
**Pull request by deining**
This PR improves the signature of various functions.

Closes #98.

### CODE REVIEW

File: content/en/functions/slicestr.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ menu:
   docs:
     parent: "functions"
 keywords: [strings]
-signature: ["slicestr STRING START [END]"]
+signature:
+  - "slicestr STRING START [END]"
+  - "strings.SliceStr STRING START [END]"
```
Comment (jmooring) [inline content/en/functions/slicestr.md:15]:
```suggestion
  - "strings.SliceString STRING START [END]"
```

File: content/en/functions/truncate.md [hunk 2]
```diff
@@ -10,7 +10,11 @@ menu:
   docs:
     parent: "functions"
 keywords: [strings]
-signature: ["truncate SIZE INPUT", "truncate SIZE ELLIPSIS INPUT"]
+signature:
+  - "truncate SIZE INPUT"
```
Comment (jmooring) [inline content/en/functions/truncate.md:14]:
```suggestion
  - "truncate SIZE [ELLIPSIS] INPUT"
```

File: content/en/functions/truncate.md [hunk 3]
```diff
@@ -10,7 +10,11 @@ menu:
...
+  - "strings.Truncate SIZE ELLIPSIS INPUT"
```
Comment (jmooring) [inline content/en/functions/truncate.md:15]:
```suggestion
  - "strings.Truncate SIZE [ELLIPSIS] INPUT"
```

File: content/en/functions/truncate.md [hunk 4]
```diff
@@ -10,7 +10,11 @@ menu:
...
+  - "truncate SIZE INPUT"
```
Comment (jmooring) [inline content/en/functions/truncate.md:16]:
```suggestion
```

File: content/en/functions/truncate.md [hunk 5]
```diff
@@ -10,7 +10,11 @@ menu:
...
+  - "strings.Truncate SIZE ELLIPSIS INPUT"
```
Comment (jmooring) [inline content/en/functions/truncate.md:17]:
```suggestion
```

**Review by jmooring — CHANGES_REQUESTED** [review]
Thanks for this. I will merge once you've made the suggested changes. At some point in the coming months, the function section of the documentation will become data driven, so please don't spend any more time adding signatures. I've added them in the past, but at this point the effort of writing (and reviewing!) isn't worth the return.

File: content/en/functions/slicestr.md (hunk 1 above)
Comment (deining) [reply content/en/functions/slicestr.md:15]: Changed as requested.

File: content/en/functions/truncate.md (hunk 2 above)
Comment (deining) [reply content/en/functions/truncate.md:14]: Brackets added as requested.

File: content/en/functions/truncate.md (hunk 3 above)
Comment (deining) [reply content/en/functions/truncate.md:15]: Brackets added as requested.

**Pull request merged**